phac-nml/irida-next
# Data exports table enhancements

## What does this PR do and why?
Adding pagination, filtering, and sorting to data exports table.
Fixes #758.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/09fa1bb3-de09-4571-91e5-eb96a991dfac)

## How to set up and validate locally
1. Navigate to the data exports page.
2. Verify sorting works for all the columns except actions.
3. Verify searching for a data export works.
4. Verify pagination works.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
